graphics/usd/Portfile

```diff
@@ -0,0 +1,218 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           compiler_blacklist_versions 1.0
+PortGroup           cmake 1.1
+PortGroup           active_variants 1.1
+
+github.setup        PixarAnimationStudios USD 19.11 v
+name                usd
+categories          graphics
+platforms           darwin
+license             Apache-2
+maintainers         @jasonliu--
+homepage            http://www.openusd.org
+description         universal scene description
+long_description    Universal Scene Description (USD) is a software \
+                    library, developed by Pixar Animation Studios, \
+                    that provides a system for authoring, reading, and \
+                    streaming time-sampled scene description for \
+                    interchange between computer graphics \
+                    applications. It organizes data into hierarchical \
+                    namespaces of primitives and their properties. \
+                    This provides a set of schemas upon which common \
+                    3-D computer graphics concepts such as geometry, \
+                    shading, models, and assets can be organized and \
+                    manipulated.
+
+checksums           rmd160  6ecba2960ecb802dd8db62344582e2d01234f3d7 \
+                    sha256  6cbd3f7be424fcb8dcc35d07f0d2cc8bc0601b2c46d06553290e6df7c5b5ec77 \
+                    size    37563045
```

Comment:
If you want to use an automatically-generated tarball, as you're doing here by not specifying `github.tarball_from`, please specify `github.tarball_from archive`. The checksums will change when you do this.

Reply:
Done.